parts/linux/cloud-init/artifacts/ubuntu/cse_install_ubuntu.sh

```diff
@@ -49,6 +49,10 @@ installDeps() {
     fi
 
     for apt_package in apt-transport-https ca-certificates ceph-common cgroup-lite cifs-utils conntrack cracklib-runtime ebtables ethtool fuse git glusterfs-client htop iftop init-system-helpers inotify-tools iotop iproute2 ipset iptables nftables jq libpam-pwquality libpwquality-tools mount nfs-common pigz socat sysfsutils sysstat traceroute util-linux xz-utils netcat dnsutils zip rng-tools kmod gcc make dkms initramfs-tools linux-headers-$(uname -r); do
+      if [[ "${apt_package}" == "fuse" && "${OSVERSION}" == "22.04" ]]; then
```

Comment:
we already special case ubuntu 22 for blobfuse above. why not reuse that?

this also adds a check for every single apt package if it's fuse and ubuntu which feels unnecessary...

Reply:
discussed offline and addressed with refactor